### PR TITLE
Fix cohere link

### PR DESCRIPTION
**Pull request by collindutter**
- [x] I have read and agree to the contributing guidelines for [submitting new pull requests](https://github.com/griptape-ai/griptape?tab=readme-ov-file#submitting-pull-requests).

## Describe your changes
Removes incorrect reference to Cohere generate API.
## Issue ticket number and link

----
📚 Documentation preview 📚: https://griptape--994.org.readthedocs.build//994/

### CODE REVIEW

File: docs/griptape-framework/drivers/prompt-drivers.md [hunk 1]
```diff
@@ -56,15 +56,10 @@ print(result.value)
 
 Griptape offers the following Prompt Drivers for interacting with LLMs.
 
-!!! warning
-    When overriding a default Prompt Driver, take care to ensure you've updated the Structure's configured Embedding Driver as well. If Task Memory isn't needed, you can avoid compatibility issues by setting `task_memory=None` to disable Task Memory in your Structure.
-
```
Comment (collindutter) [inline docs/griptape-framework/drivers/prompt-drivers.md:61]: This was more of an issue when `off_prompt` defaulted to `True`

File: docs/griptape-framework/drivers/prompt-drivers.md [hunk 2]
```diff
@@ -160,14 +153,12 @@ agent.run("Artificial intelligence is a technology with great promise.")
 
 ### Cohere
 
-The [CoherePromptDriver](../../reference/griptape/drivers/prompt/cohere_prompt_driver.md) connects to the Cohere [Generate](https://docs.cohere.ai/reference/generate) API.
+The [CoherePromptDriver](../../reference/griptape/drivers/prompt/cohere_prompt_driver.md) connects to the Cohere [Chat](https://docs.cohere.com/docs/chat-api) API.
```
Comment (dylanholmes) [inline docs/griptape-framework/drivers/prompt-drivers.md:156]: There it is ✅

File: docs/griptape-framework/drivers/prompt-drivers.md [hunk 3]
```diff
@@ -56,15 +56,10 @@ print(result.value)
...
-!!! warning
-    When overriding a default Prompt Driver, take care to ensure you've updated the Structure's configured Embedding Driver as well. If Task Memory isn't needed, you can avoid compatibility issues by setting `task_memory=None` to disable Task Memory in your Structure.
-
 ### OpenAI Chat
 
-!!! info
-    This driver uses [OpenAi function calling](https://platform.openai.com/docs/guides/function-calling) when using [Tools](../tools/index.md). You can change this to use Griptape's own tool calling implementation by setting `use_native_tools` to `False`.
-
 The [OpenAiChatPromptDriver](../../reference/griptape/drivers/prompt/openai_chat_prompt_driver.md) connects to the [OpenAI Chat](https://platform.openai.com/docs/guides/chat) API.
+This driver uses [OpenAi function calling](https://platform.openai.com/docs/guides/function-calling) when using [Tools](../tools/index.md).
```
Comment (dylanholmes) [inline docs/griptape-framework/drivers/prompt-drivers.md:62]: Does tools/index.md call out `use_native_tools=False` as an option?

**Review by dylanholmes — DISMISSED** [review]
Nice work, just one question

File: docs/griptape-framework/drivers/prompt-drivers.md (hunk 3 above)
Comment (collindutter) [reply docs/griptape-framework/drivers/prompt-drivers.md:62]: [It does now](https://github.com/griptape-ai/griptape/pull/994/files#diff-968a4b3b3f733a9b6996cfd71b083d7b50a68dd8db9bf8ea3ba6f9cafc091aff), take another gander

**Review by dylanholmes — APPROVED** [review]

**Pull request merged**